geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSUnionIntegrationTest.java

```diff
@@ -32,16 +38,17 @@
 
 import org.apache.geode.redis.ConcurrentLoopingThreads;
 import org.apache.geode.redis.RedisIntegrationTest;
-import org.apache.geode.test.awaitility.GeodeAwaitility;
 
 public abstract class AbstractSUnionIntegrationTest implements RedisIntegrationTest {
   private JedisCluster jedis;
-  private static final int REDIS_CLIENT_TIMEOUT =
-      Math.toIntExact(GeodeAwaitility.getTimeout().toMillis());
+  private static final String setKey1 = "{user1}setKey1";
+  private static final String[] setMembers1 = {"one", "two", "three", "four", "five"};
+  private static final String nonExistentSetKey = "{user1}nonExistentSet";
+  private static final String setKey2 = "{user1}setKey2";
```

Comment:
Just to avoid potential confusion, could we use a different hash tag on the keys here? The tag has nothing to do with users, just to which slot keys map to, so there's a potential for confusion with this naming. Maybe just "{tag}setKey1" etc. to keep it simple?

Reply:
Yes! A lot of my previous tests use {user1} as a tag. Should I go ahead and change everything with this commit since its quick, create a new ticket, or just fix this and from now on use a different tag?